gatsby/src/docs/accounts/sso/azure-sso.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Azure Active Directory SSO'
+sidebar_order: 1
+---
+
+## Installation
+
+1. Login to the [Azure portal](https://portal.azure.com/).
+
+2. (You can skip to (3) if you don't need to create a permission group for Sentry.) Under 
+Manage Azure Active Directory (the shield picture), click "View", which takes
+you to the Organization Overview page. On the left hand navigation, click "Groups". Create a 
+new group, add an owner, and members to the group. Set "Group type" to Office 365. For more 
+details about group creation, see the
+[Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+
+3. On the left hand navigation, click "Groups". Create a new group, add an owner, and
+members to the group. Set "Group type" to Office 365. For more details about group creation, see the
+[Azure docs](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-groups-create-azure-portal).
+
+4. Return to the Overview page. On the left hand navigation, click "Enterprise Applications".
+Then, select "Non-gallery application" and create one for Sentry. Once created, you'll
```

Comment:
"create one for Sentry" is confusing - we should be explicit about what is being created.